modules/installation-and-upgrade/pages/general-requirements.adoc

```diff
@@ -4,7 +4,6 @@
 Before you begin installation, ensure that you have:
 
 * Current {scc} organization credentials
-* Access to installation media
```

Comment:
Why do you remove this one? You still need access to the media of the Server extension be it via SCC or using an ISO image.

Reply:
This list item was referencing the unified installer and installation medium.

In the deployment guides we provide the task for connecting to SCC during installation on SLE micro 5.5 or Leap Micro 5.5, the input of reg keys etc.

The general requirements guide is not the place for this. (Or am I mistaken?)